### PR TITLE
Only read `keywords` property from the package.json manifest and not origami.json or bower.json

### PR DESCRIPTION
**Pull request by JakeChampion**
This was agreed by the team via this proposal -- https://github.com/Financial-Times/origami/pull/119

It has already been added to the draft v2 spec -- https://github.com/Financial-Times/origami-website/pull/343

### CODE REVIEW

**Comment by JakeChampion** [comment]
This should only be merged when we stop working on origami spec v1 components.

**Comment by JakeChampion** [comment]
Should this even ever get merged? 🤔

File: models/version.js [hunk 1]
```diff
@@ -251,21 +251,14 @@ function initModel(app) {
...
 			},
 
-			// Get keywords for the version, falling back through different manifests
+			// Get keywords for the version
 			keywords() {
 				const manifests = this.get('manifests') || {};
 				let keywords = [];
 
-				// Order: origami, package, bower
-				if (manifests.origami) {
```
Comment (notlee) [inline models/version.js:260]: We probably want to keep this so keywords are captured when we release to previous major versions?

Comment (notlee) [reply models/version.js:260]: ...what about non-js/npm projects? Perhaps this is why it's in origami.json

Comment (notlee) [reply models/version.js:260]: ...we have few of those, all non-components, so I actually think that's ok but don't remember that reasoning coming up in the proposal

Comment (notlee) [reply models/version.js:260]: >We probably want to keep this so keywords are captured when we release to previous major versions?

Oh I just saw your comment pointing this out :D

Comment (notlee) [reply models/version.js:260]: I would be happy to not merge this. Repo data will continue to support v1 components and the worse that will happen is v2 components will have too many keywords. We should probably lint v2 components to make sure they are not setting these properties in origami.json using build tools

**Pull request closed without merging**